### PR TITLE
Move Distributor interface back to runtime.

### PR DESCRIPTION
**Pull request by ozevren**
Distributor is an interface consumed by the runtime package.

### CODE REVIEW

**Comment by ozevren** [comment]
/test e2e-simpleTests
--

File: galley/pkg/runtime/state.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ type State struct {
 
 	distribute  bool
 	strategy    *publish.Strategy
-	distributor publish.Distributor
+	distributor Distributor
```
Comment (nmittler) [inline galley/pkg/runtime/state.go]: Can we make it `distributor` as well so it won't leak out of the package?

Comment (ozevren) [reply galley/pkg/runtime/state.go]: It is meant to be passed in from the outside.

**Pull request merged**